src/components/NewChapterForm.tsx

```diff
@@ -0,0 +1,269 @@
+import React from "react";
+import { z } from "Zod";
+import { useForm, SubmitHandler } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { ChapterRequest } from "src/app/api/chapter-request/route.schema";
+import { ErrorMessage } from "@hookform/error-message";
+
+const NewChapterForm = () => {
+  const title = [
+    "First Name",
+    "Last Name",
+    "University Email",
+    "Phone Number",
+    "University",
+    "University Address",
+  ];
+
+  // const schema = ChapterRequest
+  // .object({
+  //   firstName: ChapterRequest.string().required(),
+  //   lastName: ChapterRequest.string().required(),
+  //   universityEmail: ChapterRequest.string().email("This is not a valid email").required(),
+  //   phoneNumber: ChapterRequest.string().length(10, "Phone number must be 10 digits").required(),
+  //   university: ChapterRequest.string().required(),
+  //   universityAddress: ChapterRequest.string().required(),
+  //   leadershipExperience: ChapterRequest.string().required(),
+  //   motivation: ChapterRequest.string().required(),
+  //   availabilities: z.string(),
+  //   questions: z.string(),
+  // })
+  // .required()
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<ValidationSchema>({
+    resolver: zodResolver(ChapterRequest),
+  });
+
+  const onSubmit: SubmitHandler<typeof ChapterRequest> = (data) =>
```

Comment:
Should be `SubmitHandler<z.infer<typeof ChapterRequest>>`, or equivalently, `SubmitHandler<ValidationSchema>` since you defined it below